test/e2e/framework/test_context.go

```diff
@@ -421,6 +424,9 @@ func AfterReadingAllFlags(t *TestContextType) {
 			t.Host = defaultHost
 		}
 	}
+	if len(t.BearerToken) == 0 {
+		t.BearerToken = rand.String(12)
```

Comment:
given the tests allow remote hosts, we should use crypto/rand and a token with more bits:
```
tokenBytes := make([]byte, 16)
if _, err := rand.Read(tokenBytes); err != nil {
	panic("could not generate random bearer token")
}	
t.BearerToken = hex.EncodeToString(tokenBytes)
```

Reply:
I agree we could use a token with more bits, but currently apiserver only listen on localhost:
```go
	o.SecureServing.BindAddress = net.ParseIP("127.0.0.1")
	// Disable insecure serving
	o.InsecureServing.BindPort = 0
```

Reply:
ok, i was confusing this with `TestContext.Host`.

maybe we should add this comment just in case:

```
// if support for listening on external addresses is added, the BearerToken should be randomized with
// a cryptographically secure algorithm - e.g. crypto/rand.
o.SecureServing.BindAddress = net.ParseIP("127.0.0.1")
```

Reply:
comment added